static/templates/subscription_setting_icon.handlebars

```diff
@@ -2,6 +2,8 @@
     <div class="flex">
         {{#if invite_only}}
         <i class="fa fa-lock" aria-hidden="true"></i>
+        {{else if is_web_public}}
```

Comment:
I realise there is a bug in linter somewhere which makes it indifferent to `else if`. I will look around if I can find it and fix it along with violations in a separate commit later on.